man/fido_cred_set_authdata.3

```diff
@@ -163,6 +167,16 @@ is zero, the extensions of
 are cleared.
 .Pp
 The
+.Fn fido_cred_set_blob
+function sets the cred blob to be stored with
```

Comment:
either `.Dq credBlob`, or `FIDO 2.1 credential blob` if you prefer

Reply:
fixed in ff5b0b66b67aa6f336f84db06a3c20ddd87872fc